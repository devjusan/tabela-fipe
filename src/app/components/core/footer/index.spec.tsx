import { render, cleanup, screen } from '@testing-library/react';
import Footer from '.';

describe('Footer', () => {
  afterAll(() => {
    cleanup();
  });

  it('should match snapshot', () => {
    const footerText = 'Tabela Fipe';
    render(<Footer />);

    expect(screen.getByText(footerText)).toMatchSnapshot();
  });
});
