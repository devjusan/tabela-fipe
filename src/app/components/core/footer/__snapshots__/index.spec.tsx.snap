// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should match snapshot 1`] = `
<p
  class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
>
  Tabela Fipe 
</p>
`;
